spec/models/like_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe Like, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'Comment Likes Validations' do
+    subject { Like.new(author_id: 1, post_id: 1) }
+
+    before { subject.save }
+
```

Comment:
Please add a unit test for `#update_likes_counter` of the Like model as it's required to test custom model methods.